### DISCUSSION TITLE
Xenon bootloader update from 0.5.0 to 0.6.1 not working?

### DISCUSSION THREAD

**Discussion by ScruffR** [adafruit/Adafruit_nRF52_Bootloader#240] [Q&A]
**Set up**
  - Particle Xenon
  - Windows 10 (21H1 Build 19043-1202)
  - SW on the device (bootloader mode)
```
UF2 Bootloader 0.5.0 lib/nrfx (v2.0.0) lib/tinyusb (0.9.0-22-g7cdeed54) lib/uf2 (remotes/origin/configupdate-9-gadbb8c7)
Model: Particle Xenon
Board-ID: nRF52840-Xenon-v1
SoftDevice: S140 version 6.1.1
Date: Apr  4 2021
```

**Describe the bug**

I did expect that I can replace the current bootloader by putting the device in bootloader mode and copying the `update-particle_xenon_bootloader-0.6.1_nosd.uf2` file to the device. 
However when the device comes back ist still reports bootloader 0.5.0

Is it not supposed to work as UF2 upload or is that file incompatible with the Xenon?

**Comment by hathach** [comment]
it should work as other boards, but I don't have a xenon to test with.

**Comment by ScruffR** [comment]
@hathach , thanks for the reply :+1:

So you also think that my process was correct and should work?

 - plug in device
 - put device into bootloader mode (i.e. press/hold MODE button and tap RESET)
 - wait for device's onboard LED to turn green and expose BOOT drive (i.e. XENONBOOT) on host computer
 -  copy .UF2 file to BOOT drive
 - wait for device to restart

**Comment by hathach** [comment]
Yes, your procedure is correct

**Reply by ScruffR** [reply]
In that case I don't understand why it doesn't work.

**Reply by hathach** [reply]
I have no clues as well, probably something specific to the board. Only Adafruit boards are officially supported ( I also don't have this board as well). Therefore you got to do your own work by either trying to 
- insert log/printf and recompile bootloader and check to see where it fails
- update bootloader using CDC interface to see if that works.
